runtime/src/configs.rs

```diff
@@ -17,11 +17,8 @@ pub mod asset_config;
 pub use asset_config::AssetType;
```

Comment:
We have an hybrid situation here where some configs are in this file and other in the configs/ folder. Let's choose one approach ?

Reply:
Fine